src/geometry/stress.hpp

```diff
@@ -98,7 +98,7 @@ class Stress
   private:
     Simulation_context& ctx_;
 
-    Density& density_;
+    const Density& density_;
```

Comment:
usually it is `T const& t`, not `const T& t`.

Reply:
Changed to `T const&`, still need to go over the other files.